client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-conflicts.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeConflicts extends Component {
+	placeholder = () => {
+		return (
+			<div>
+				<div className="domain-connect__is-placeholder">
+					<span></span>
+					<span></span>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { isPlaceholder, conflictingRecords } = this.props;
```

Comment:
Yeah. It seemed like an odd choice for a variable name to me too, but so many other components seem to use `isPlaceholder` that I just followed along for consistency. I'm not sure if it's that important for this component since its likely that it won't be reused anywhere else in the application. I do think that something like `isLoading` is more semantic.